run/GCHP/HISTORY.rc.templates/HISTORY.rc.carbon

```diff
@@ -56,16 +56,16 @@ COLLECTIONS: 'SpeciesConc',
              'Emissions',
              #'Metrics',
              #'Budget',
-             'Carbon',
              #'CloudConvFlux',
              #'ConcAfterChem',
              #'FV3Dynamics',
              #'GCHPctmEnvLevCenter',
              #'GCHPctmEnvLevEdge',
              #'KppDiags',
+             #'ProdLoss',
              #'RxnConst',
              #'RxnRates',
-             'StateMet',
+             #'StateMet',
```

Comment:
Isn't this needed for carbon benchmarks?

Reply:
Now fixed in e043fbd7